src/Ldap.php

```diff
@@ -1283,6 +1283,57 @@ public function delete($dn, $recursively = false)
     }
 
     /**
+     * Delete single attributes from an LDAP-Node
+
+     * This method removes single attributes from a node identified by
+     * <var>$dn>/var>. The attributes have to be given as array where the
+     * array-key is the attribute-name and the array-value is the attribute-value
+     * that is to be removed.
+     *
+     * To remove multiple entries of an attribute pass an array with the values
+     * to be removed as value of the key.
+     *
+     * So if you want to remove more than one 'memberUid'-attribute you would
+     * pass ```array('memberUid' => ['uid1', 'uid2',...]);``` as <var>$attributes</var>
+     *
+     * Beware that passing an empty array will remove <strong>all</strong> entries
+     * of the attribute. Therefore you'Ll have to set the <var>$emptyAll</var>-flag!
+     *
+     *
+     * @param       $dn
+     * @param array $attributes
+     * @param bool  $emptyAll
+     *
+     * @return self
+     */
+    public function deleteAttributes($dn, array $attributes, $emptyAll = false)
+    {
+        // Safety-flap: Check whether there are empty arrays that would cause
+        // complete removal of entries without the emptyAll flag.
+        foreach ($attributes as $key => $value) {
+            if ($emptyAll !== true && empty($value)) {
+                unset($attributes[$key]);
+            }
+        }
+
+        if ($dn instanceof Dn) {
+            $dn = $dn->toString();
+        }
+
+//        $resource = $this->getResource();
+
+        ErrorHandler::start(E_WARNING);
```

Comment:
I just copied the behaviour of the other methods in the class that all have the ```ErrorHandler``` around the calls to the native ```ldap_*```-functions. Not sure whether they are necessary